apps/authentication-service/src/shared/users/usersService.test.ts

```diff
@@ -0,0 +1,62 @@
+import { http, HttpResponse } from "msw";
+
+import { User } from "@/shared/users/usersModel";
+import { AUTH0_USER_BY_ID } from "@/test/handlers";
+import {
+  exampleClientID,
+  exampleUserAlreadyHasAClientError,
+  exampleUserID,
+  exampleUserInfoResponse,
+  exampleUserWithClientId,
+} from "@/test/testData/users";
+import { server } from "@/test/testServer";
+import { getUserClientId, getUserById, setUserClientId } from "./usersService";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+jest.mock("auth0", () => ({
+  ...jest.requireActual("auth0"),
+  UserInfoClient: class {
+    getUserInfo = (_token: string) => ({ data: exampleUserInfoResponse });
+  },
+}));
+
+describe("User Service tests", () => {
```

Comment:
I wrap each function in it's own describe.

`describe("User Service tests", () => {
    describe("getUserById", () => {
      it("", ( => {
`